pkg/config/config.go

```diff
@@ -347,7 +347,8 @@ const (
 
 // Executors contains executors configuration parameters.
 type Executors struct {
-	Plugins Plugins `yaml:",inline" koanf:",remain"`
+	DisplayName string  `yaml:"displayName"`
```

Comment:
Can you add displayName to the `values.yaml` file for the Botkube Helm chart, so that we have the display names by default for all executor config groups? Thanks!

Reply:
I've added to specific executor, cos sometimes in the group there is more the one plugin. Please verify if it's ok.